src/org/zaproxy/zap/extension/fieldenumeration/resources/Messages.properties

```diff
@@ -0,0 +1,3 @@
+fieldenumeration.field.popup=Field Enumeration
+fieldenumeration.desc	= Identify blacklist characters
```

Comment:
How about something more verbose?
> An extension to help testers identify which characters are accepted for targeted parameters/fields.

Or something like that. I haven't used it yet but hopefully since you've built it you can come up with a good description of what the extension does.